elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -139,5 +182,15 @@ $LOCAL-VARIABLES: (
 			height: pfe-local(size, $region: close);
 			width: 	pfe-local(size, $region: close);
 		}
+
+		:host([pfe-media]) & {
+			top: 0px; 
+			right: 0;
+			border-radius: 3px;
+			padding: 7px;
```

Comment:
This style `padding: 7px;` and this one `--pfe-modal__close--Padding: 10px;` are conflicting.  I'd recomment:

```
padding: pfe-local(Padding, $region: close, $fallback: 10px);
```

Reply:
thanks, fixed!